astropy/units/tests/test_quantity_array_methods.py

```diff
@@ -124,8 +122,6 @@ def test_swapaxes(self):
         assert q_swapaxes.unit == q.unit
         assert np.all(q_swapaxes.value == q.value.swapaxes(0, 2))
 
-    @pytest.mark.xfail(sys.byteorder == 'big' and NUMPY_LT_1_22,
```

Comment:
I think we should we change this to `NUMPY_LT_1_21_1`, because presumably it still fails on all older versions of numpy.

Reply:
I thought about it but we only run this job weekly on `main`. Is that overkill? And if not, should I put `NUMPY_LT_1_21_1` in `utils` or just this test module? We do not have a precedent to do such comparison on bugfix release, do we?

Reply:
I'm thinking more of someone actually using a big-endian machine, installing astropy, and then running the tests. Would be nice if they do not get a failure!

As for location, the advantage of putting NUMPY_LT_1_21_1 in `utils` is that we'll see it when we get to `numpy >= 1.22`...

Reply:
Okay, I updated the patch as you suggested. Now we just have to wait a few hours...

Reply:
Thank you for the quick review!